### DISCUSSION TITLE
RTL support / disable swiping & dragging / async loading

### DISCUSSION THREAD

**Discussion by wisniowski** [fancyapps/ui#163] [Q&A] [answered]
Hey guys,

I owe the commercial license & I am so glad it works with pure JS instead of jQuery. Although, I have a few issues:
1. How do I enable the RTL support? Is that possible?
2. Is there a way to disable swiping & dragging?

**Comment by fancyapps** [comment, marked as answer]
Hi,

Sorry, RTL support is on the TODO list.

To disable vertical dragging, set `dragToClose: false`.

To disable touch events for main Carousel:

```JS
Carousel : {
    Panzoom: {
      touch: false
    }
}
```
To disable dragging/panning of the image:

```JS
Image: {
  touch: false
}
```

**Reply by wisniowski** [reply]
Thanks!
